content/web/tutorials/standalone-example.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Standalone Example-Web WorldWind/NASA WorldWind"
+date: 2017-11-15T14:33:47-08:00
+draft: false
+header: Standalone Example
+listdescription: Describes the how to host Web WorldWind on your own HTTP server.
+---
+
+## Hosting Locally
+
+---
+
+### The Data
+
+Get the [latest release](https://github.com/NASAWorldWind/WebWorldWind/releases/tag/v0.9.0-RC2) on GitHub.
```

Comment:
We'll want links going to https://github.com/NASAWorldWind/WebWorldWind/releases/latest